src/plays/color-palettes-generator/components/utils.js

```diff
@@ -0,0 +1,11 @@
+function componentToHex(c) {
+  var hex = c.toString(16);
+
+  return hex.length == 1 ? '0' + hex : hex;
+}
+
+function rgbToHex(r, g, b) {
+  return '#' + componentToHex(r) + componentToHex(g) + componentToHex(b);
```

Comment:
@AbhiPatel10 , it would be great if add comments for the functions,

Reply:
File removed